packages/shared/ReactFeatureFlags.js

```diff
@@ -172,7 +172,7 @@ export const disableInputAttributeSyncing = false;
 // Filter certain DOM attributes (e.g. src, href) if their values are empty
 // strings. This prevents e.g. <img src=""> from making an unnecessary HTTP
 // request for certain browsers.
-export const enableFilterEmptyStringAttributesDOM = false;
+export const enableFilterEmptyStringAttributesDOM = __EXPERIMENTAL__;
```

Comment:
why do the flag change in this PR?

Reply:
It was just not on before, where as it probably should've been so we can test this behavior in OSS.